OpenKeychain/src/androidTest/java/org/sufficientlysecure/keychain/ui/AsymmetricFileOperationTests.java

```diff
@@ -79,8 +79,8 @@
 import static org.sufficientlysecure.keychain.matcher.CustomMatchers.withSignatureNone;
 
 
-@RunWith(AndroidJUnit4.class)
-@LargeTest
+//@RunWith(AndroidJUnit4.class)
```

Comment:
Please include a comment in all broken tests why they are disabled. I we merge this pr, people will be confused why some tests are commented out. At least put a line on top which states that this tests needs to be fixed with a TODO